src/placement/jump_map.c

```diff
@@ -1243,13 +1247,13 @@ jump_map_obj_find_reint(struct pl_map *map, struct daos_obj_md *md,
 	 */
 	allow_status = PO_COMP_ST_UPIN | PO_COMP_ST_DOWN | PO_COMP_ST_DRAIN;
 	D_INIT_LIST_HEAD(&reint_list);
-	rc = obj_layout_alloc_and_get(jmap, &jop, md, allow_status, &layout,
-				      NULL, NULL);
+	rc = obj_layout_alloc_and_get(jmap, &jop, md, allow_status, reint_ver,
+				      &layout, NULL, NULL);
 	if (rc < 0)
 		D_GOTO(out, rc);
 
 	allow_status |= PO_COMP_ST_UP;
-	rc = obj_layout_alloc_and_get(jmap, &jop, md, allow_status,
+	rc = obj_layout_alloc_and_get(jmap, &jop, md, allow_status, reint_ver,
```

Comment:
the reint_ver seems same as md.omd_ver? not sure why need both two parameters of md and reint_ver for jump_map_obj_find_reint() etc.? if there are same, can obj_layout_alloc_and_get just use md.omd_ver?

Reply:
hmm, it looks clearer to explicitly specify the rebuild version for find_reint() version, while md are suppose to be the object md here.  Not sure if it should be put together, I will think about it. Thanks.